src/containers/Tenant/Query/i18n/en.json

```diff
@@ -15,5 +15,10 @@
 
   "preview.title": "Preview",
   "preview.not-available": "Preview is not available",
-  "preview.close": "Close preview"
+  "preview.close": "Close preview",
+
+  "method-description.script": "YQL Script – for YQL-scripts combining DDL and DML. API call: schema.scripting",
+  "method-description.scan": "Scan – read-only queries, potentially reading a lot of data. API call: table.ExecuteScan",
+  "method-description.data": "Data – DML queries for changing and fetching data in serialization mode. API call: table.executeDataQuery",
+  "method-description.query": "YQL – QueryService – any queries. An experimental API call supposed to replace all existing methods. API Call: query.ExecuteScript"
```

Comment:
Do we really need to duplicate query modes titles (YQL Script, Scan, Data, YQL - QueryService) in help popover?